client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -140,13 +160,35 @@ class TransferDomainPrecheck extends React.PureComponent {
 		);
 	}
 
+	getTransferRestrictionMessage() {
+		const { domain } = this.props;
+		const {
+			creationDate,
+			loading,
+			termMaximumInYears,
+			transferEligibleDate,
+			transferRestrictionStatus,
+		} = this.state;
+
+		return (
```

Comment:
`transferIsRestricted` if it's not `loading`, so we don't need the `loading` code in that component.

Reply:
Yep. Only display the component if `! loading`.